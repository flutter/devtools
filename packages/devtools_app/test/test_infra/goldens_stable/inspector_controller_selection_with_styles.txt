▼[R][root]
  ▼[M]MyApp
    ▼[M]MaterialApp
      ▼[S]Scaffold
      ├───▼[C]Center
      │     [/icons/inspector/textArea.png]Text <-- selected
      └─▼[A]AppBar
          [/icons/inspector/textArea.png]Text
