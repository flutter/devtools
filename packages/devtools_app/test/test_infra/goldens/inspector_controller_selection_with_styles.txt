▼[R][root]
  ▼[M]MyApp
    ▼[M]MaterialApp
      ▼[S]Scaffold
      ├───▼[C]Center
      │     [T]Text <-- selected
      └─▼[A]AppBar
          [T]Text
